scripts/webhook-prepare-kustomize.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script prepares fpga_plugin ready for installation by
+#
+# kubectl apply -k deployments/fpga_plugin/overlays/af
+#
+# or
+#
+# kubectl apply -k deployments/fpga_plugin/overlays/region
+
+service="intel-fpga-webhook-svc"
+secret="intel-fpga-webhook-certs"
+
+script_dir="$(realpath $(dirname $0))"
+srcroot="$(realpath ${script_dir}/..)"
+
+kustomize_secret_dir="${srcroot}/deployments/fpga_admissionwebhook/base/${secret}-secret"
```

Comment:
Would be nice to put generated files to `.gitignore`.
BTW, is it possible to have these files generated in `${srcroot}/${secret}-secret`, so that a user could notice the new folder?

Reply:
Good point. Added the created secrets directory to `.gitignore`.

Unfortunately the secret directory cannot be brought to the root level due to kustomize security restrictions: all included files - except for `bases` - must be on the same level or deeper in the directory structure than `kustomization.yaml` that uses them. This issue has troubled many, and a bunch of improvements has been proposed during past months, without a breakthrough so far.